plugins/importexport/crossref/templates/settings.tpl

```diff
@@ -34,9 +35,12 @@
 			<li>{translate key="plugins.importexport.crossref.error.publisherNotConfigured" publisherUrl=$publisherUrl}</li>
 		{/if}
 
-	</ul>
 	{else}
-		{translate key="plugins.importexport.crossref.requirements.satisfied"}
+		<li>{translate key="plugins.importexport.crossref.requirements.satisfied"}</li>
+		{if $currentJournal->getSetting('onlineIssn')|escape || $currentJournal->getSetting('printIssn')|escape}
+			<li>{translate key="plugins.importexport.crossref.requirements.issnWarning" issnUrl=$issnUrl}</li>
+		{/if}
+	</ul>
```

Comment:
It looks good to me - the indent for the <li> above looks kind of weird, but that's only because it's indented to visually clarify the if statement.